Doc/library/http.server.rst

```diff
@@ -444,6 +444,15 @@ the following command uses a specific directory::
 .. versionadded:: 3.7
     ``--directory`` argument was introduced.
 
+By default, the server supports HTTP up to version HTTP/1.0. The option ``-p/--protocol``
```

Comment:
Should this say «up to»?  HTTP/0.9 had no headers I think, so a server with HTTP/1.0 really needs clients that support 1.0.   (I haven’t checked the code recently)

Reply:
@merwok `BaseHTTPRequestHandler` does support HTTP/0.9, even when you configure it to be conformant with HTTP/1.0 or HTTP/1.1. But I agree that ‘up to’ was unnecessary. I have just updated the PR.